src/UIComponents/DataDisplay.jsx

```diff
@@ -64,29 +83,46 @@ class DataDisplay extends Component {
   };
 
   render() {
-    const { data, setCurrentColumn, currentPanel } = this.props;
+    const {
+      data,
+      setCurrentColumn,
+      setColumnRef,
+      setHighlightColumn,
+      currentPanel
+    } = this.props;
 
     return (
       <div id="data-display" style={styles.panel}>
-        <div style={styles.statement}>
+        {/*<div style={styles.statement}>
           Predict{" "}
           <span style={styles.statementLabel}>
             {this.props.labelColumn || "..."}
           </span>
-          {currentPanel === "dataDisplayFeatures" && (
-            <span>
-              {" "}
-              based on{" "}
-              <span style={styles.statementFeature}>
-                {this.props.selectedFeatures.length > 0
-                  ? this.props.selectedFeatures.join(", ")
-                  : ".."}
-              </span>
-              {"."}
+          <span>
+            {" "}
+            based on{" "}
+            <span style={styles.statementFeature}>
+              {this.props.selectedFeatures.length > 0
+                ? this.props.selectedFeatures.join(", ")
+                : ".."}
             </span>
+            {"."}
+          </span>
+        </div>*/}
```

Comment:
can we delete this?

Reply:
I will do so.  I think we should be able to show it on subsequent screens though, to remind the user what they are predicting...